fast-enrichment-service/proto/validation_service.proto

```
@@ -0,0 +1,134 @@
+syntax = "proto3";
+
+package gpp.g3.validation;
```

Buf (1.54.0)

Files with package "gpp.g3.validation" must be within a directory "gpp/g3/validation" relative to root but were in directory "fast-enrichment-service/proto".

(PACKAGE_DIRECTORY_MATCH)